core/services/ardupilot_manager/mavlink_proxy/AbstractRouter.py

```diff
@@ -109,6 +109,9 @@ def add_endpoint(self, endpoint: Endpoint) -> None:
         if endpoint in self._endpoints:
             raise ValueError("Endpoint already exists.")
 
+        if endpoint.name in [endpoint.name for endpoint in self._endpoints]:
```

Comment:
you can also do: 
```suggestion
        if any(lambda name: endpoint.name == name, self._endpoints):
```

Reply:
Yeah
I'm really having mixed feelings with the lambda functions. It's syntax is very bad.

Reply:
`any(condition, list)` has a clear concept over mapping a list to extract names and checking if a name exist inside a list.

Reply:
The concept is clear. The lambda syntax that is not :cry:

Reply:
Javascript got it so right with the arrow functions...